app/assets/stylesheets/application.css.scss

```diff
@@ -15,24 +15,36 @@ header {
   flex-flow: row;
   background: purple;
   color: white;
-  
+
+  #navBar {
+    display: block;
+    padding: 0.4em;
+    float: left;
+  }
+
   #logo {
     display: block;
     padding: 0.4em;
+    float: left;
```

Comment:
Don't use float inside a flexbox. You might need to add an expandable spacer between the right and left. Something like `.spacer { flex: 1 }`.